tests/e2e/specs/editor/publishingFlow.js

```diff
@@ -78,6 +81,22 @@ describe('Publishing Flow', () => {
       await page.evaluate(() =>
         wp.data.dispatch('core/edit-post').toggleFeature('welcomeGuide')
       );
+
+      await page.reload();
```

Comment:
Is the full page reload really needed? :/

Reply:
It was copied from [here](https://github.com/WordPress/gutenberg/blob/c31555d4cec541db929ee5f63b900c6577513272/packages/e2e-test-utils/src/create-new-post.js#L37-L63). 

It is only one reload, it gutenberg is doing it, it must be for a reason.